test/tools/spec-runner/index.js

```diff
@@ -774,6 +861,9 @@ function testOperation(operation, obj, context, options) {
       const cursor = obj[operationName].apply(obj, args);
       opPromise = cursor.toArray();
     } else {
+      if (!obj[operationName]) {
+        throw new Error(`Unknown operation "${operationName}"`);
+      }
```

Comment:
This was too useful while making these changes to not leave it in the PR, even though I assume that this is legacy code